sentry_streams/sentry_streams/pipeline/pipeline.py

```diff
@@ -147,10 +147,20 @@ class Step:
 
     name: str
     ctx: Pipeline
+    config: Optional[Any] = field(default=None, init=False)
 
     def __post_init__(self) -> None:
         self.ctx.register(self)
 
+    def override_config(self, app_config: Any) -> Any:
+        """
+        A step's config in the application would be overriden by
+        its config value loaded from the file
```

Comment:
remove the mention to the file. This method does not need to know where the config overrides are coming from.